.github/workflows/create_llvm.yml

```diff
@@ -4,13 +4,15 @@ on:
   push:
     branches:
       - main
+    paths:
+      - '.github/workflows/create_llvm.yml'
```

Comment:
Just making sure I understand correctly, this means this workflow *only* runs when a push to `main` touches this file?

Reply:
yes, that's right or we run the workflow manually (but if we want to update we'd need to manually delete the caches).

Reply:
Thanks, seems like that makes sense.